hocon.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name        = 'hocon'
-  s.version     = '1.1.2'
+  s.version     = '1.2.0'
```

Comment:
We do want to have a CI job for hocon, probably a good idea to change it to work this way